src/tools/s2wasm.cpp

```diff
@@ -33,6 +34,7 @@ int main(int argc, const char *argv[]) {
   bool generateEmscriptenGlue = false;
   std::string startFunction;
   std::vector<std::string> archiveLibraries;
+  bool validate = false;
```

Comment:
OK, how about this: I'm looking at one error right now. If it's the only one, I'll add it to this PR. Otherwise I'll do them separately :)